### ISSUE TITLE
Package win32_gui not analysed

### ISSUE THREAD

**Issue by sigurdm** [dart-lang/pub-dev#6971]
From the worker-logs:
```
2023-08-19 15:36:05.472823 [pub_worker.process_payload] SHOUT: ERROR: HTTP error, status = 400, body: <?xml version='1.0' encoding='UTF-8'?><Error><Code>EntityTooLarge</Code><Message>Your proposed upload is larger than the maximum object size specified in your Policy Document.</Message><Details>Content-length exceeds upper bound on range</Details></Error>, #0      upload.<anonymous closure> (package:pub_worker/src/upload.dart:45:9)
```

Seems the upload is larger than the max. We should probably not crash here, but upload the logs, and a message that dartdoc was too large (and by how much).

Should we increase the max-size?

**Comment by gmpassos** [comment]
The `win32_gui` package has only 58 KB, BTW.

**Comment by gmpassos** [comment]
It's a FireBase alert:

https://www.codejam.info/2023/04/firebase-functions-entity-too-large.html

**Comment by gmpassos** [comment]
I have published a new version of `win32_gui`, but it is still not being analyzed.

**Comment by gmpassos** [comment]
[win32](https://pub.dev/packages/win32) still not analyzed

**Comment by isoos** [comment]
I think this can now be closed, we've migrated to dartdoc 6.3.0 and now the generated content is within the sanity limits.
